services/dqa-secondary-reports/report_parser.py

```diff
@@ -66,20 +71,24 @@ def parse(self):
             # Data-Quality/Dictionary/DCC_DQA_Dictionary.csv
             # to ensure consistensy and no typos
 
-            site_record = {
-                'site': self.site,
-                'rank': rank
+            record = {
+                'finding': finding,
+                'prevalence': prevalence,
+                'rank': rank,
+                'response': response,
+                'cause': cause
             }
 
+            site_record = copy.copy(record)
```

Comment:
So you know, mutable data structures such as `dict`s and `list`s have more idiomatic ways of creating copies:

```python
# dict
d2 = d1.copy()
d2 = dict(d1)

# list
l2 = l1[:]
l2 = list(l1)
```

Note that these are *shallow* copies. If any of the values in the structure are mutable and mutated later, they will be reflected in all copies.